### DISCUSSION TITLE
Korio - Problem getting file stats on iOS

### DISCUSSION THREAD

**Discussion by martinguay** [korlibs/korge#1811] [Q&A] [answered]
Hi all!
I'm currently using korIO and I have the following code to retrieve stats from a local file saved on the device:
```
val file = LocalVfs[getAbsolutePath(filePath)]
if (file.exists()) {
    val stat = file.stat()
    println(stat.createTime)
    println(stat.lastAccessTime)
    println(stat.modifiedTime)
}
```

On Android, this code is working perfectly and I'm able to get all the infos from the stats. However, on iOS, the createTime, lastAccessTime and modifiedTime are always equal to 0. The only thing in the stats that seems to be properly initialized is the file size.

Is something missing in this code to work properly on iOS?
Thank you very much!

**Comment by soywiz** [comment, marked as answer]
Fixed here: https://github.com/korlibs/korge/discussions/1811
